platform/install/configure/config.mdx

```diff
@@ -93,6 +93,30 @@ config:
       X-XSS-Protection: 1; mode=block
 ```
 
+### Setting additional certificate authority for Platform
+If you decide to configure vCluster Platform helm release with  `additionalCA`, it will be automatically used by all vClusters managed by the vCluster Platform to establish a connection between vClusters and the Platform. You don't need to tweak individual vCluster configuration to enable this.
+This will enable secure communication between your vClusters and the vCluster Platform.
+```yaml
+additionalCA: "(base64 encoded CA)"
+```
+
+Also, if you need to import externally managed vCluster to the vCluster Platform, you can also enable a secure connection by grabbing the value from the  existing vCluster Platform helm release, you can do this by running:
```

Comment:
We don't call installing vCluster Platform as the `vCluster Platform helm release`, but this something to figure out how we want to refer to it. 

For now, I would call it vCluster Platform until we standardize it.

Reply:
ok, removed `helm release`